pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -475,6 +475,20 @@ self: super: {
       };
     });
 
+  vimacs = super.vimacs.overrideAttrs(old: {
+    buildPhase = ''
+      substituteInPlace bin/vim \
+        --replace '/usr/bin/vim' 'vim' \
+        --replace '/usr/bin/gvim' 'gvim'
+      rm -r plugin/vimacs
```

Comment:
The original upstream project duplicates the `bin/vim` script in a second file `vimacs/plugin/vimacs/vim`.  Since Nix can correctly manipulate the user paths, there is no need to have two copies.

Reply:
Please also add that explanation as an inline comment. Everything that is non-obvious should be commented inline, the next person editing the derivation might not know that.

Reply:
Done